pkg/database/workout_type.go

```diff
@@ -16,14 +16,15 @@ const (
 	WorkoutTypeSnowboarding WorkoutType = "snowboarding"
 	WorkoutTypeSwimming     WorkoutType = "swimming"
 	WorkoutTypeKayaking     WorkoutType = "kayaking"
+	WorkoutTypeGolfing      WorkoutType = "golfing"
 )
 
 func WorkoutTypes() []WorkoutType {
-	return []WorkoutType{WorkoutTypeRunning, WorkoutTypeCycling, WorkoutTypeWalking, WorkoutTypeSkiing, WorkoutTypeSnowboarding, WorkoutTypeSwimming, WorkoutTypeKayaking}
+	return []WorkoutType{WorkoutTypeRunning, WorkoutTypeCycling, WorkoutTypeWalking, WorkoutTypeSkiing, WorkoutTypeSnowboarding, WorkoutTypeSwimming, WorkoutTypeKayaking, WorkoutTypeGolfing}
 }
 
 func DistanceWorkoutTypes() []WorkoutType {
```

Comment:
Is this actually a distance-based workout? Do you track distance while golfing? Real, honest question, I have no idea!

Reply:
yes, basically it's the distance that you walk on the course between each ball hit.